code/modules/antagonists/eldritch_cult/knowledge/ash_lore.dm

```diff
@@ -70,7 +70,6 @@
 	name = "Curse of blindness"
 	gain_text = "Blind man walks through the world, unnoticed by the masses."
 	desc = "Curse someone with 2 minutes of complete blindness by sacrificing a pair of eyes, a screwdriver and an object that the victim has touched with their bare hands."
-	cost = 1
```

Comment:
Do not remove `cost`.

Reply:
Thanks. Must have removed it by mistake when fixing conflicts.